addons/website/static/tests/tour_utils/widget_lifecycle_patch_wysiwyg.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import Wysiwyg from "web_editor.wysiwyg";
+import { patch } from "@web/core/utils/patch";
+
+const patchWysiwyg = () => patch(Wysiwyg.prototype, "widget_lifecycle_patch_wysiwyg.wysiwyg", {
```

Comment:
What's the point in creating a function for this if this is to call it only once just after its definition? :thinking: